exporters/prometheus/exporter.go

```diff
@@ -0,0 +1,243 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build go1.18
+// +build go1.18
+
+package prometheus // import "go.opentelemetry.io/otel/exporters/prometheus"
+
+import (
+	"context"
+	"sort"
+	"strings"
+	"unicode"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/internal/global"
+	"go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/metric/metricdata"
+)
+
+// Exporter is a Prometheus Exporter that embeds the OTel metric.Reader
+// interface for easy instantiation with a MeterProvider.
+type Exporter struct {
+	metric.Reader
+	Collector prometheus.Collector
+}
+
+// collector is used to implement prometheus.Collector.
+type collector struct {
+	metric.Reader
+}
+
+// config is added here to allow for options expansion in the future.
+type config struct{}
+
+// Option may be used in the future to apply options to a Prometheus Exporter config.
+type Option interface {
+	apply(config) config
+}
+
+// NewExporter returns a Prometheus Exporter.
+func NewExporter(options ...Option) Exporter {
+	reader := metric.NewManualReader()
```

Comment:
This relies on the default temporality selector always return cumulative. Can we add a comment here to help developers understand this?